bin/dcps_tests.lst

```diff
@@ -426,7 +426,7 @@ tests/FACE/Compiler/idl_test1_main/run_test.pl: !DCPS_MIN !WCHAR !NO_BUILT_IN_TO
 tests/FACE/Compiler/idl_test3_main/run_test.pl: !DCPS_MIN !WCHAR !NO_BUILT_IN_TOPICS
 tests/FACE/Compiler/idl_test_fixed/run_test.pl: !DCPS_MIN !WCHAR !NO_BUILT_IN_TOPICS
 
-tests/DCPS/XTypes/run_test.pl: !DCPS_MIN
+tests/DCPS/XTypes/run_test.pl --type-consistency: !DCPS_MIN
```

Comment:
Does it need to have the `--type-consistency` command line argument if that's the only way it's ever invoked?